Extension/package.json

```diff
@@ -135,7 +135,8 @@
           "type": "string",
           "enum": [
             "Enabled",
-            "Disabled"
+            "Disabled",
+            "MissingIncludesOnly"
```

Comment:
Perhaps `EnabledWhenIncludesResolve`?  `MissingIncludesOnly` sounds like only `#include` squiggles will ever be shown.